### PR TITLE
refactor: update invest/withdraw page with flat design

### PR DESCRIPTION
**Pull request by BartDeCaluwe**
This PR will:

- update the invest/withdraw  page layout

## Checklist

- [ ] I used the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
- [ ] Unit tests have been created if a new feature was added.
- [ ] Translations have been added and extracted if a new feature was added.

### CODE REVIEW

File: src/components/LiquidityPoolInteraction.tsx [hunk 1]
```diff
@@ -31,78 +29,20 @@ interface LiquidityPoolInteractionProps {
...
-		form.setFieldsValue({
-			amount: getMaxValue,
-		});
-	};
-
-	const getMaxValue = action === "invest" ? maxInvestmentAmount : maxWithdrawalAmount;
 
 	const validateMaxAmount = (value): Promise<void> => {
-		const maxValue = getMaxValue;
 		const validationMessage = `'amount' needs to be less than or equal to ${
```
Comment (RuvendeGroote) [inline src/components/LiquidityPoolInteraction.tsx]: intl

Comment (RuvendeGroote) [reply src/components/LiquidityPoolInteraction.tsx]: same for min validation

File: src/components/LiquidityPoolInteraction.tsx [hunk 2]
```diff
@@ -122,62 +62,43 @@ export const LiquidityPoolInteraction = ({
...
+				className="bg-credix-primary"
+				placeholder="USDC"
+				labelClassName="mb-4"
+				lang="en"
+				type="number"
+				required={true}
+				rules={[
+					{ required: true },
+					{
+						validator(_, value) {
```
Comment (RuvendeGroote) [inline src/components/LiquidityPoolInteraction.tsx]:
```
validator: (_, value) => validateMaxAmount(value)
```
is nicer imo

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

File: src/components/LiquidityPoolInteraction.tsx [hunk 3]
```diff
@@ -122,62 +67,62 @@ export const LiquidityPoolInteraction = ({
...
+				label={intl.formatMessage(MESSAGES.amountInputLabel)}
+				step="0.1"
+				className="bg-credix-primary"
+				placeholder={intl.formatMessage(MESSAGES.amountInputPlaceholder)}
+				labelClassName="mb-4"
+				lang="en"
+				type="number"
+				required={true}
+				rules={[
+					{ required: true },
```
Comment (RuvendeGroote) [inline src/components/LiquidityPoolInteraction.tsx]: does this need a message?

**Review by RuvendeGroote — CHANGES_REQUESTED** [review]

File: src/components/LiquidityPoolInteraction.tsx (hunk 3 above)
Comment (BartDeCaluwe) [reply src/components/LiquidityPoolInteraction.tsx]: It technically doesn't as it has a default message. But I've added one for better i18n support.

**Review by RuvendeGroote — APPROVED** [review]

**Pull request merged**